client/pom.xml

```diff
@@ -96,5 +96,10 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents.client5</groupId>
+            <artifactId>httpclient5</artifactId>
+            <version>5.5</version>
```

Comment:
This dependency should be specified in the root pom.xml.  Then here the dependency will omit the version